config/initializers/rack_attack.rb

```diff
@@ -102,6 +102,32 @@ def headers
       end
     end
 
+    ### Prevent SMS and voice spam ###
+
+    # A user can use the `/otp/send` path to spam phone numbers with SMSs and
+    # voice calls.
+
+    # Throttle SMS and voice transactions by IP address
+    #
+    # Key: "rack::attack:#{Time.now.to_i/:period}:otps/ip:#{req.remote_ip}"
+    if AppConfig.env.otps_per_ip_track_only_mode == 'true'
+      track(
+        'otps/ip',
+        limit: AppConfig.env.otps_per_ip_limit.to_i,
+        period: AppConfig.env.otps_per_ip_period.to_i,
+      ) do |req|
+        req.remote_ip if req.path == /\/otp\/send/
+      end
+    else
+      throttle(
+        'otps/ip',
+        limit: AppConfig.env.otps_per_ip_limit.to_i,
+        period: AppConfig.env.otps_per_ip_period.to_i,
+      ) do |req|
+        req.remote_ip if req.path =~ /\/otp\/send/
```

Comment:
same as above, suggest `%r{}` for readability

```suggestion
        req.remote_ip if req.path =~ %r{/otp/send}
```